src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -56,6 +56,7 @@
       <Link>System\SpanTestHelpers.cs</Link>
     </Compile>
     <Compile Include="CompareToTests.cs" />
+    <Compile Include="System\Runtime\CompilerServices\CallerArgumentExpressionAttributeTests.cs" />
```

Comment:
This needs to be in `Condition="'$(TargetGroup)' != 'netfx'` section of the file since this attribute does not exist on netfx.